### PR TITLE
addings docs

### PR DESCRIPTION
**Pull request by aluedeke**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- Java
+You can use those standalone or with your Selenium/Appium based tests. Support for other frameworks like Cypress, StoryBooks, Playwright are on the Roadmap.
+
+The best way to integrate Sauce Visual into your existing tests (or writing new ones) is to follow the examples listed [in this repo](../README.md).
+
+After you have executed your tests you will find your Visual test results in the Builds History Page:
+
+![Diff History Page}](./resources/BuildHistoryPage.png)
+
+By selecting one of the builds you get to the Diff Review Page where you cann Approve/Reject detected diffs.
```
Comment (marstupa) [inline docs/README.md:20]: Typo here - "cann"

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Visual Beta Documentation
+
+## Intro
+
+Sauce Visual helps you to compare uploaded images (so called snapshots) against reference images (called baselines). It offers APIs to upload snapshots and compare those against baselines. The result of the comparison prrocess is a diff with details about eventually detected differences. All snapshots in Sauce Visual need to be assigned to visual builds.
```
Comment (marstupa) [inline docs/README.md:5]: Typo: prrocess

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+## Workflow
+
+To be able to benfit from Sauce Visual you typically add it to your exsting automated tests using provided libraries. We currently provide bindings for:
```
Comment (marstupa) [inline docs/README.md:9]: Typos: benefit, existing

File: docs/README.md (hunk 2 above)
Comment (marstupa) [reply docs/README.md:5]: Also wondering if keywords that they will encounter on the UI might be in bold or code formatted - stuff like snapshot, baseline, diff

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+![Diff Review Page}](./resources/DiffReviewPage.png)
+
+## Visual Statuses
+
+Visual uses different Statuses:
+
+| Status   | Description |
+| -------- | ----------- |
+| In Progress | Those builds have been finished yet. As soon as this happend they can be reviewed |
```
Comment (marstupa) [inline docs/README.md:31]: typo: happened

File: docs/README.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+| No Changes | The snapshots assigned to your build where equal to your baseline. This is considered as susccess as your assertions succeeded. |
+| For Review | There where either no baselines available to compare against your upload snapshot or some of them where different then their baselines. You are supposed to review those detected diffs. As long as those changes aren't accepted they are considered a failure state. |
+| Accpeted   | All detected changes where accepted. This is considered a success state. |
```
Comment (marstupa) [inline docs/README.md:34]: Typo: Accepted

File: docs/README.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+![Diff Review Page}](./resources/DiffReviewPage.png)
+
+## Visual Statuses
+
+Visual uses different Statuses:
+
+| Status   | Description |
+| -------- | ----------- |
+| In Progress | Those builds have been finished yet. As soon as this happend they can be reviewed |
+| No Changes | The snapshots assigned to your build where equal to your baseline. This is considered as susccess as your assertions succeeded. |
```
Comment (marstupa) [inline docs/README.md:32]: Typos: 
build were equal
success

**Review by marstupa — CHANGES_REQUESTED** [review]

**Review by paweltomaszewskisaucelabs — APPROVED** [review]
LGTM

File: docs/README.md (hunk 2 above)
Comment (paweltomaszewskisaucelabs) [reply docs/README.md]: It's interesting idea. Personally I would prefer to keep bold style for links to other resources.

Comment (paweltomaszewskisaucelabs) [reply docs/README.md]: @aluedeke WYT?

**Review by marstupa — APPROVED** [review]

**Pull request merged**